com.unity.render-pipelines.universal/Shaders/PostProcessing/FinalPost.shader

```diff
@@ -60,58 +42,15 @@ Shader "Hidden/Universal Render Pipeline/FinalPost"
             float2 positionNDC = uv;
             int2   positionSS  = uv * _SourceSize.xy;
 
-            half3 color = Load(positionSS, 0, 0).xyz;
+            #if _POINT_SAMPLING
+            half3 color = SAMPLE_TEXTURE2D_X(_SourceTex, sampler_PointClamp, uv).xyz;
+            #else
+            half3 color = SAMPLE_TEXTURE2D_X(_SourceTex, sampler_LinearClamp, uv).xyz;
+            #endif
```

Comment:
This seems like an overkill to duplicate variants for sampler state. Is there a way to configure sampler / filtering mode on RTHandle?

Reply:
I completely agree. :( Unfortunately I didn't see a way to set sampler state from the command buffer API? It seems like you can only set textures and the sampler state is fused to the texture which means that changing it as part of the final pass could potentially affect other parts of the frame since it's changing persistent state on the object rather than specifying a per-draw parameter?

Reply:
Ok. That's unfortunate. Then I guess we should at bare least strip the unused variants from build (look at ShaderPreprocessor.cs)